### ISSUE TITLE
Callout box styling tweaks

### ISSUE THREAD

**Issue by AmeliaWellers** [matthewcrist/ctoec#87]
### Callout box layout
**Field layout**
One thing we noticed while walking through it today was it is a little clunky if you're trying to go fast. So what about...

*Layout (Horizontal/Card) — If possible, make "horizontal" default
Title
Content
Icon On/Of
Icon Search*

Please remove the second instance of "View a full list of available icons at https://ctoec.org/oec-icons", as this info is also in context for the icon search.

Changes to "TITLE" style in callout box:

font-size: 1.5rem;
margin-bottom: 1rem;

**Error Message**: Change error message for mistyping an icon name in the search. It currently says "Invalid Post". Should be something like "No icon matches found"

### Visual Builder

- Make the icon preview much smaller, fx: 50x50px
- Make the "title" read out in the visual builder BOLD. Alert box showed as an H3 

### Styling Adjustments
Styling adjustments for callouts based on most recent CTDS values and some custom tweaks (pulled from STAGE ticket)

Purple Bar on Horizontal + Card:
border-left-width (or top-width on card): .375rem solid #61366F
border-radius: .25rem;
Horizontal Margin:

margin-bottom: .275rem;
Card Margin:

margin-bottom: .5rem;
Shadow for both:

box-shadow: .1rem, .25rem 1.25 rem rgba(97,54,111,.1)
Icons:

color: #61366F
– SVG width and max-height: 2.5rem

**Comment by bloom-hondo** [comment]
> Field layout
> One thing we noticed while walking through it today was it is a little clunky if you're trying to go fast. So what about...
> 
> Layout (Horizontal/Card) — If possible, make "horizontal" default
> Title
> Content
> Icon On/Of
> Icon Search

Fields are reordered

> Please remove the second instance of "View a full list of available icons at https://ctoec.org/oec-icons", as this info is also in context for the icon search.

@AmeliaWellers I'm not seeing this. Can you post a screenshot?

> Error Message: Change error message for mistyping an icon name in the search. It currently says "Invalid Post". Should be something like "No icon matches found"

We can't change the error message.

**Comment by AmeliaWellers** [comment]
Thanks Matt. I've attached the screenshot below - now that the fields are reordered, the duplicate textplanations are more noticable. 

![Screen Shot 2021-03-02 at 10.07.21 AM.png](https://images.zenhubusercontent.com/600707487eacdcc9a535ffa4/2fb70bf3-18e3-40a6-8f2d-1a9b2929883d)

**Comment by AmeliaWellers** [comment]
### Callout box layout
 - @bloom-hondo I refreshed all the caches, but I didn't see the proposed changes for the field layout reorg and removing the duplicate sentence.
 - As discussed today, the error message can't change, so we'll keep it as is. 
 - @admoorgit, I had copied and pasted this part, but upon rereading, am not sure what "Changes to "TITLE" style in callout box" is referencing. Is it a duplicate of what's been fixed in the visual builder?

### Visual Builder
This part looks great to me -  The icon size is much more manageable, the bolded title helps, and it's nice to confirm the CSS styling within the editor. This will make training a lot easier. 

### Styling Adjustments
@admoorgit you are the better choice for QAing this section.

**Comment by AmeliaWellers** [comment]
### Field Layout take 2
- Checked out production, where the changes are. The layout makes more intuitive sense now. Shouldn't need to change this again, and I'll check back in after seeing how cc's respond to it. 
- The  "View a full list of available icons at https://ctoec.org/oec-icons" is still present (see screenshot above) Suggestion: remove the sentence at the very bottom of the list, and keep the longer version that is below the "Search Icons" field in each callout.

**Comment by admoorgit** [comment]
Hello @Amelia! Here's a response to your question:

> @admoorgit, I had copied and pasted this part, but upon rereading, am not sure what "Changes to "TITLE" style in callout box" is referencing. Is it a duplicate of what's been fixed in the visual builder?

This is proposed styling changes for the output from the field marked "Title" in the callout box block. I'm glad you pointed it out, because I went back to look and actually didn't address line-height, which I just did. @bloom-hondo the new values are in the Figma file here: https://www.figma.com/file/YifRehO7BrZhP1Wdru67Zo/CTDS-OEC?node-id=0%3A1

One thing I **think** we talked about but don't see it logged here, so I want to make sure we log it: bullets in callout boxes — any way to get those indented properly? I seem to recall the answer was no, but not why we can't...

- a.
